hydra-node/test/Hydra/NetworkSpec.hs

```diff
@@ -52,8 +54,14 @@ spec = do
     prop "can roundtrip CBOR encoding/decoding of Hydra Message" $ prop_canRoundtripCBOREncoding @(Message SimpleTx)
     roundtripAndGoldenSpecs (Proxy @(Message SimpleTx))
 
+  describe "Message Persistence" $ do
```

Comment:
The test case name indicates ("it throws..") that this is a unit test about a specific function. I would suggest to use exactly that function name in the `describe` scope. This helps in finding relevant tests and eases future renaming of this function.

```suggestion
  describe "configureMessagePersistence" $ do
```